index.html

```diff
@@ -0,0 +1,102 @@
+<!DOCTYPE html>
+<html>
+    <head>
+        <meta charset="UTF-8">
+        <meta name="viewport" content="width=device-width", initial-scale="1.0">
+        <title>Html-capstone</title>
+        <link rel="stylesheet" href="styles.css">
+    </head>
+    <body>
+        <section class="main-section">
+        <header>
+            <ul class="social-icons">
+                <li><a href="#" class="header-icon"><img src="assets/face.png" alt="the facebook logo"></a></li>
+                <li><a href="#" class="header-icon"><img src="assets/twitter.svg" alt="the twitter logo"></a></li>
+                <li><a href="#" class="header-icon">English</a></li>
+                <li><a href="#" class="header-icon">Homepage</a></li>
+                <li><a href="#" class="header-icon">Logout</a></li>
+            </ul>  
```

Comment:
Kindly wrap this and all other navigation menus in the nav tag. This is good for SEO and accessibility features :+1: